docker-compose.yml

```diff
@@ -17,13 +17,13 @@ services:
     image: rethinkdb
     ports:
       - "58585:8080"
-      - "28015"
+      - "28015:28015"
   bdb-server:
     build: ./compose/server
     environment:
       BIGCHAINDB_DATABASE_HOST: rdb
       BIGCHAINDB_API_ENDPOINT: http://bdb-server:9984/api/v1
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
     ports:
-      - "9984"
+      - "32675:9984"
```

Comment:
not pinning the ports was intentional, since when working with the server there may be clashes, hence not pinning is the safest to avoid clashes